nuancier/lib/__init__.py

```diff
@@ -231,7 +231,7 @@ def edit_election(session, election, election_name, election_folder,
         election.election_folder = election_folder
         edited.append('election folder')
 
-    if election.election_year != election_year:
+    if election.election_year != int(election_year):
```

Comment:
This feels kinda wrong, I think the data that is passed here should already be an int.

Reply:
https://github.com/fedora-infra/nuancier/blob/8a62363c3a652ecacf5f11c4fa9e4ab12554cf65/nuancier/forms.py#L100

Maybe it will be better to render it as an [IntegerField](https://wtforms.readthedocs.io/en/stable/fields.html#wtforms.fields.IntegerField) instead of converting it in edit_election? What do you think?

Reply:
This sounds like a much better fix indeed!
Thanks